database/seeders/LaratrustSeeder.php

```diff
@@ -497,5 +497,9 @@ public function run() {
             'permission_id' => 13,
             'role_id' => 15
         ]);
+        DB::table('permission_role')->insert([
+            'permission_id' => 7,
+            'role_id' => 15
+        ]);
```

Comment:
What role are we adding here?

Reply:
permission_id 7 = files
role_id 15 = marketing